### ISSUE TITLE
Cargo.Lock invalid?

### ISSUE THREAD

**Issue by ts468** [myfreeweb/evscript#1]
I wanted to package `evscript` for `NixOS`, but it seems as `cargo vendor` failes with:

```
error: failed to sync

Caused by:
  found duplicate version of package `libc v0.2.34` vendored from two sources:

	source 1: https://github.com/rust-lang/libc#7c1297c9
	source 2: registry https://github.com/rust-lang/crates.io-index
```

Unfortunately, I'm no Rust user myself, and so I wondered whether you could maybe update the `Cargo.Lock` for me, please? `NixOS` requires a valid `Cargo.Lock` as otherwise it could not guarantee the required reproducibility of a build.

**Comment by valpackett** [comment]
Yeah you can't vendor it right now, I'm currently using the git master version of `nix` (the crate, not related to the package manger :D) which pulls the git master version of libc. I guess I can try cargo's patch thing, but really it's a bit early to package this, I haven't even released 0.1.0.

**Comment by ts468** [comment]
Thanks, and indeed, it's really a bit early to package evscript. But it just looks like a very interesting project and I had a few configurations that I wanted to try using it with. It would be nice if you could maybe just ping me once you've released a first version. I'll try to package it up for `nix` (this time the package manager :) then!
